src/app/app-routing.module.ts

```diff
@@ -0,0 +1,53 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { UIRouterModule } from '@uirouter/angular';
+import { Transition } from '@uirouter/core';
+import { PeopleService } from 'src/people.service';
+import { uiRouterConfigFn } from 'src/router.config';
+import { AboutComponent } from './about/about.component';
+import { HelloComponent } from './hello/hello.component';
+import { PeopleComponent } from './people/people.component';
+import { PersonComponent } from './person/person.component';
+
+const helloState = { name: "hello", url: "/hello", component: HelloComponent };
+const aboutState = { name: "about", url: "/hello", component: AboutComponent };
```

Comment:
`about` and `hello` states have exactly the same URLs. Is it intended or is it a typo? :)